akka-projection-jdbc/src/it/scala/akka/projection/jdbc/JdbcContainerOffsetStoreSpec.scala

```diff
@@ -99,7 +99,11 @@ object JdbcContainerOffsetStoreSpec {
 
   object OracleSpecConfig extends ContainerJdbcSpecConfig("oracle-dialect") {
     val name = "Oracle Database"
-    override def newContainer() =
+
+    // related to https://github.com/testcontainers/testcontainers-java/issues/2313
+    // otherwise we get ORA-01882: timezone region not found
+    System.setProperty("oracle.jdbc.timezoneAsRegion", "false")
```

Comment:
If you see this and is asking yourself what that is, rest assured, that's pure art. It's normal to not understand it at first. 

(This is needed for running the Oracle container in GH Actions. Not needed for Travis nor locally 🤷 )